required/latex-lab/latex-lab-tikz.dtx

```diff
@@ -198,27 +198,31 @@
    \tag_mc_end:
 %    \end{macrocode}
 % this is the code that calculates and sets the BBox attribute
+% \changes{0.80d}{2025-09-27}{Support also pgfplots. Tagging issue 924} 
 %    \begin{macrocode}
    \cs_set:Npn\pgfqpoint##1##2
     {
-      \dim_to_decimal_in_bp:n {##1+ \pgf@picminx}
-      \c_space_tl
-      \dim_to_decimal_in_bp:n {##2+ \pgf@picminy}
-      \c_space_tl
-      \dim_to_decimal_in_bp:n {##1+ \pgf@picmaxx}
-      \c_space_tl
-      \dim_to_decimal_in_bp:n {##2+ \pgf@picmaxx}
+      \tl_set:Ne\l_tmpa_tl
+       {
+         \dim_to_decimal_in_bp:n {##1+ \pgf@picminx}
+         \c_space_tl
+         \dim_to_decimal_in_bp:n {##2+ \pgf@picminy}
+         \c_space_tl
+         \dim_to_decimal_in_bp:n {##1+ \pgf@picmaxx}
+         \c_space_tl
+         \dim_to_decimal_in_bp:n {##2+ \pgf@picmaxx}
+       }  
     }
    \cs_if_exist:cT { pgf@sys@pdf@mark@pos@pgfid\the\pgf@picture@serial@count }
     {
+      \use:c { pgf@sys@pdf@mark@pos@pgfid\the\pgf@picture@serial@count }
       \tag_struct_gput:ene
         {\tag_get:n{struct_num}}
         {attribute}
         {
           /O /Layout /BBox~
           [
-            \use:c
-             { pgf@sys@pdf@mark@pos@pgfid\the\pgf@picture@serial@count }
+            \l_tmpa_tl         
```

Comment:
am I supposed to understand why the code changed here from what it was before?

Perhaps a line or tw of documentation what is going on there so that others (and perhaps you at some time in the future) understand  this?

Reply:
with tikz the command is expandable and could be used directly in the attribute, with pgfplots we have to go a detour ... I extended the documentation a bit.